src/main/java/uk/gov/justice/digital/hmpps/whereabouts/services/CellService.kt

```diff
@@ -0,0 +1,16 @@
+package uk.gov.justice.digital.hmpps.whereabouts.services
+
+import org.springframework.stereotype.Service
+import uk.gov.justice.digital.hmpps.whereabouts.dto.CellMoveDetails
+import uk.gov.justice.digital.hmpps.whereabouts.dto.CellMoveResult
+
+@Service
+class CellService(val prisonApiService: PrisonApiService) {
```

Comment:
Mostly because the first thing that came to mind was phone service - could this be named `CellMoveService` ?

Reply:
Ah yes, it could actually.